README.md

```diff
@@ -1 +1,44 @@
-# MultiDID
+# multidid
+
+> A very compact representation of DIDs and DID URLs
+
+Multidid is a representation strategy for DIDs and DID URLs that is very compact and extensible. It allows any DID method to be represented as a string of bytes.
+
+## Format
+
+```
+<multidid-code><method-code><method-bytes><url-length><url-bytes>
+```
+
+Where:
+
+* `multidid-code` - the value `0x0d1d` encoded as a varint
+* `method-code` - a varint encoded multicode for the [DID Method](https://www.w3.org/TR/did-core/#a-simple-example)
+* `method-bytes` - data representing the method-id
+* `url-length` - a varint describing the length of the `url-bytes` parameter
+* `url-bytes` - a *UTF-8* encoded string representing the [DID URL parameters](https://www.w3.org/TR/did-core/#did-url-syntax)
+
+## Method multicodes
+
+By default any DID method can be represented using the `0x00` code as the `method-code`. Any DID method can add an additional, more optimized, representation by submitting a specification to the `methods` folder in this repository, claiming a prefix in the [multicodec table](https://github.com/multiformats/multicodec/blob/master/table.csv), and adding it to the method table below.
```

Comment:
```suggestion
By default any DID method can be represented using the `0x55` code as the `method-code`. Any DID method can add an additional, more optimized, representation by submitting a specification to the `methods` folder in this repository, claiming a prefix in the [multicodec table](https://github.com/multiformats/multicodec/blob/master/table.csv), and adding it to the method table below.
```

I think you mean this here

Reply:
I think you instead of registering those codes in https://github.com/multiformats/multicodec/blob/master/table.csv it may be a better idea to have table in this repo instead. Since multidid's are prefixed with umbrella `0x0d1d` we don't need to register other codes but that alone.

Reply:
This is in fact what we are doing. See the table in the readme.

Reply:
👍 I was just speaking to the

> claiming a prefix in the [multicodec table](https://github.com/multiformats/multicodec/blob/master/table.csv), and adding it to the method table below.